plugins/woo-ai/src/product-description/product-description-button-container.tsx

```diff
@@ -161,21 +185,33 @@ export function WriteItForMeButtonContainer() {
 			includedProps.push( name );
 		} );
 
-		return [
+		const instructions = [
```

Comment:
I re-introduced the ability to append instructions as it seemed like the cleanest way to inject things like "tone of voice" into the prompt.

Reply:
👍